website/static/js/projectorganizer.js

```diff
@@ -1,220 +1,1023 @@
+/**
+ * Handles Project Organizer on dashboard page of OSF.
+ * For Treebeard and _item API's check: https://github.com/caneruguz/treebeard/wiki
+ */
 'use strict';
 
 var Handlebars = require('handlebars');
 var $ = require('jquery');
-var HGrid = require('hgrid');
+var m = require('mithril');
+var Treebeard = require('treebeard');
 var bootbox = require('bootbox');
 var Bloodhound = require('exports?Bloodhound!typeahead.js');
 var moment = require('moment');
 var Raven = require('raven-js');
 
-require('../vendor/jquery-drag-drop/jquery.event.drag-2.2.js');
-require('../vendor/jquery-drag-drop/jquery.event.drop-2.2.js');
-require('../vendor/bower_components/hgrid/plugins/hgrid-draggable/hgrid-draggable.js');
-
 var $osf = require('osfHelpers');
 
-//
-// Globals
-//
-// ReloadNextFolder semaphores
-var reloadNewFolder = false;
-var rnfPrevItem = '';
-var rnfToReload = '';
 // copyMode can be 'copy', 'move', 'forbidden', or null.
+// This is set at draglogic and is used as global within this module
 var copyMode = null;
-var projectOrganizer = null;
 
-$.ajaxSetup({ cache: false });
+// Initialize projectOrganizer object (separate from the ProjectOrganizer constructor at the end)
+var projectOrganizer = {};
```

Comment:
@caneruguz Not sure if this object is necessary, but if we keep it should be renamed to fit its purpose, which is to namespace the 2 Bloodhound objects.